### ISSUE TITLE
Unhandled resource configuration exception

### ISSUE THREAD

**Issue by mvar** [Sylius/Sylius#7796]
When I have configuration like this:

```yml
# routing.yml
foo.bar:
    resource: |
        alias: foo.bar
        grid: foo_bar
        templates: [FooUiBundle:Crud]
    type: sylius.resource
```

I get this unhandled exception:

```
Whoops, looks like something went wrong.

1/1
ContextErrorException in Response.php line 380:
Warning: Header may not contain more than a single header, new line detected
in Response.php line 380
at Response->sendHeaders() in Response.php line 418
at Response->send() in app_dev.php line 32
at require('/Users/mantas/Sites/foo/web/app_dev.php') in router_dev.php line 40
```

This is not actual error. Handling of actual error failed and entire application crashed (printed message above). Actual error was this (not printed for the user):

```
Invalid type for path "routing.templates". Expected scalar, but got array in alias: foo.bar
grid: foo_bar
templates: [FooUiBundle:Crud]
(which is being imported from "/Users/mantas/Sites/foo/app/config/routing.yml").
```

I think this is Symfony issue and I have opened issue in Symfony project (symfony/symfony#22072) yesterday. I was not able to reproduce it on Symfony alone so I open this issue in Sylius too.

Sylius version: v1.0.0-beta.1
Symfony version: v3.2.6

**Comment by pjedrzejewski** [comment]
The exception is correct, it should be scalar value (prefix for templates, not array of templates). Please reopen if still an issue. :)

**Comment by mvar** [comment]
@pjedrzejewski second exception is correct, but as I said, it's not shown for the user. I have extracted it using xdebug. User gets very primitive exception screen of the Symfony (that gray one which occurs only on unhandled exceptions, without toolbar).

When user provides invalid resource configuration he gets "Header may not contain more than a single header, new line detected" which does not explain what happened.

**Comment by mvar** [comment]
@pjedrzejewski did you understand what is the problem I was talking about?

Just add route like this and you will see error message which does not explain anything:

```yml
foo:
    resource: |
        555
    type: sylius.resource
```

**Comment by pamil** [comment]
Anyway, it's a Symfony bug (assuming all exceptions message does not contain new line characters), not a Sylius one.

**Comment by leogout** [comment]
Hi, it's indeed a Symfony bug, and it have been [resolved in 3.2.7](https://github.com/symfony/symfony/pull/22154). Just run composer update and you should be good to go !
